ztp/gitops-subscriptions/argocd/example/optional-extra-manifest/ipsec/README.md

```diff
@@ -2,57 +2,60 @@
 
-* Enable IPsec encryption yaml
-* IPsec endpoint configuration with NMState yml
-* build.sh script that creates a MachineConfig to import external certs into IPsec NSS and apply the IPsec configuration
-
-Also available in OpenShift docs: [IPsec encryption for external traffic](https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.html#nw-ovn-ipsec-external_configuring-ipsec-ovn)
+Also available in OpenShift docs: [IPsec encryption for external traffic](https://docs.openshift.com/container-platform/4.16/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.html#nw-ovn-ipsec-external_configuring-ipsec-ovn)
 
 ### Prerequisites
 
 * `butane` utility installed, minimal version 0.20.0.
+* An IPsec endpoint certificate for each node in PKCS#12 format named the with the nodename and a CA cert in PEM format.
+Optionally, a single certificate for all the nodes (the nodes' IDs (IP, DNS, etc) should be set in the certificate SAN extension).
+* Include `enable-ipsec.yaml` file in one of the additional install-time manifests directories defined in the `extraManifests.searchPaths` field in the SiteConfig file
+More info about the `extraManifests.searchPaths` mechanism can be found in the [GitOps ZTP README](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/README.md)
+
+Below are two options to configure IPsec encryption for external traffic
+
+### IPsec configuration with NMState Operator
+
+#### Prerequisites
+
+* NMState Operator installed on the cluster.
+
+#### Import external certs
 
-* IPsec endpoint certificate in PKCS#12 format and a CA cert in PEM format.
+Use `import-certs.sh` script that creates a MachineConfig to import the external certs.
 
-### Import external certs and configure IPsec
+Replace `<node1>, <node2>, etc` with the node name.
+If you are using a single certificate replace only `<node1>` with the certificate name and delete the others.
 
-Provide the following certificate files:
+Run `./import-certs.sh` to build the MachineConfig.
 
-- `left_server.p12`: The certificate bundle for the IPsec endpoints
-- `ca.pem`: The certificate authority that you signed your certificates with
+Include `99-ipsec-master-import-certs.yaml` and `99-ipsec-worker-import-certs.yaml` files in one of the additional install-time manifests directories.
 
-Configure the following values in `ipsec-endpoint-config.yml`:
+#### Configure IPsec
 
-- `$clusterNode` - the IP address or DNS hostname of the cluster node for the IPsec tunnel on the cluster side
-- `$externalHost` - the external host IP address or DNS hostname
-- `$externalAddress` - the IP address or subnet of the external host
+Create an IPsec configuration with an NMState Operator node network configuration policy.
 
-For example:
+Include the policy template file `ipsec-config-policy.yaml` in one of the additional install-time manifests directories.
```

Comment:
I think we should add a comment here mentioning that the remote-end parameters may need to be modified, namely right and rightsubnet.